fastlane/Fastfile

```diff
@@ -46,17 +38,107 @@ platform :ios do
     # Run UI tests
     run_tests(
       scheme: "tba-ui-tests",
-      devices: ["iPhone X (11.4)", "iPhone X (12.0)"],
       derived_data_path: "fastlane/derived_data/derived_data_ui",
       test_without_building: true
     )
   end
 
+  desc "Create a new app version (major, minor, patch, or anything else for a version bump)"
+  lane :new_version do |options|
+    version_type = options[:version_type]
+    if version_type != "build"
+      increment_version_number(bump_type: version_type)
+    end
+    increment_build_number
+    commit_version_bump
+    push_to_git_remote
```

Comment:
This needs to be moved out of here - it's going to end up being a problem with that `while` loop below. We'll end up committing until we find a proper version which... isn't great. We should probably only commit if we had to bump.